TdvfPkg/TdvfPkg.dec

```diff
@@ -70,7 +70,7 @@
   ## Pcd8259LegacyModeEdgeLevel defines the default edge level for legacy
   #  mode's interrrupt controller.
   #  For the corresponding bits, 0 = Edge triggered and 1 = Level triggered.
-  gUefiTdvfPkgTokenSpaceGuid.Pcd8259LegacyModeEdgeLevel|0x0000|UINT16|0x20009
+  gUefiTdvfPkgTokenSpaceGuid.Pcd8259LegacyModeEdgeLevel|0x0000|UINT16|0x00000
```

Comment:
0x20009 is the token number of Pcd8259LegacyModeEdgeLevel. It should not be set to 0x00000

Reply:
Why? The goal is to enforce all IRQ0-23 edge triggered.

Reply:
I think there is a misunderstanding of the gUefiTdvfPkgTokenSpaceGuid.Pcd8259LegacyModeEdgeLevel|**0x0000**|UINT16|**0x20009** here.
0x20009 is the token number of Pcd8259LegacyModeEdgeLevel. 0x20009 itself should not be set to 0x00000.
The value of Pcd8259LegacyModeEdgeLevel is **0x0000**. It is correct.

Reply:
Oh I see. thanks for explanation. I'll update the patch.